terraform/workspace_variables/app_config.yml

```diff
@@ -14,13 +14,16 @@ staging:
   <<: *default
   RAILS_ENV: staging
   RACK_ENV: staging
+  SETTINGS__FEATURE_FLAGS__NEW_FILTERS: true
 
 prod:
   <<: *default
   RAILS_ENV: production
   RACK_ENV: production
+  SETTINGS__FEATURE_FLAGS__NEW_FILTERS: true
 
 sandbox:
   <<: *default
   RAILS_ENV: sandbox
   RACK_ENV: sandbox
+  SETTINGS__FEATURE_FLAGS__NEW_FILTERS: true
```

Comment:
Can remove all three additions to this file - the update in settings.yml is sufficient, only need to add an override in `app_config` if temporarily enabling the feature for QA.

Reply:
Ah cool. Thanks for spotting that @malcolmbaig. Have updated.